apps/explorer/src/components/footer/Footer.tsx

```diff
@@ -2,97 +2,64 @@
-                                Blog
-                            </Link>
-                        </li>
-                        <li>
-                            <Link href="https://github.com/MystenLabs/sui/blob/main/doc/paper/sui.pdf">
-                                Whitepaper
-                            </Link>
-                        </li>
-                    </ul>
-                </div>
-                <div>
-                    <h6>Build</h6>
-                    <ul>
-                        <li>
-                            <Link href="https://docs.sui.io/">Docs</Link>
-                        </li>
-                        <li>
-                            <Link href="https://github.com/MystenLabs">
-                                GitHub
-                            </Link>
-                        </li>
-                        <li>
-                            <Link href="https://discord.gg/sui">Discord</Link>
-                        </li>
-                    </ul>
-                </div>
-                <div>
-                    <h6>Follow</h6>
-                    <ul>
-                        <li>
-                            <Link href="https://mystenlabs.com/#community">
-                                Press
-                            </Link>
-                        </li>
-                        <li>
-                            <Link href="https://twitter.com/SuiNetwork">
-                                Twitter
-                            </Link>
-                        </li>
-                        <li>
-                            <Link href="https://www.linkedin.com/company/mysten-labs/">
-                                LinkedIn
-                            </Link>
-                        </li>
-                    </ul>
-                </div>
-                <div>
-                    <h6>Legal</h6>
-                    <ul>
-                        <li>
-                            <Link href="https://mystenlabs.com/legal?content=terms">
-                                Terms & Conditions
-                            </Link>
-                        </li>
-                        <li>
-                            <Link href="https://mystenlabs.com/legal?content=privacy">
-                                Privacy Policy
-                            </Link>
-                        </li>
+function FooterLinks({ links }: { links: FooterItems }) {
+    return (
+        <>
+            {links.map(({ category, items }) => (
+                <div
+                    key={category}
+                    className="flex flex-col gap-y-3.5 text-left"
+                >
+                    <Text variant="captionSmall/bold" color="gray-60">
+                        {category}
+                    </Text>
+                    <ul className="flex flex-col gap-y-3.5">
+                        {items.map(({ title, href }) => {
+                            return (
```

Comment:
This reminds me, we should make the implicit return consistent through eslint. Can you follow-up with this? I have a mild preference towards enforcing implicit return, but I don't care _that_ much.

Reply:
yea, I prefer as well. I think this was a case of debugging and forgetting to change code back to implicit return which is where an eslint rule would be useful.

Reply:
added eslint rule, will follow up with an auto-fix